OpenAI-DotNet/OpenAIClientSettings.cs

```diff
@@ -72,7 +74,7 @@ public OpenAIClientSettings(string domain, string apiVersion = DefaultOpenAIApiV
         /// <param name="useActiveDirectoryAuthentication">
         /// Optional, set to true if you want to use Azure Active Directory for Authentication.
         /// </param>
-        public OpenAIClientSettings(string resourceName, string deploymentId, string apiVersion = DefaultAzureApiVersion, bool useActiveDirectoryAuthentication = false)
+        public OpenAIClientSettings(string resourceName, string deploymentId, string apiVersion = DefaultAzureApiVersion, bool useActiveDirectoryAuthentication = false, TimeSpan? pooledConnectionLifetime = null)
```

Comment:
I don't think we would need to set this in the settings since it is possible to provide your own HttpClient in the constructor of the OpenAIClient itself.

Reply:
So remove the setting from this class and just default to 15 minutes where the `HttpClient` is instantiated?

Reply:
Sounds great.